### ISSUE TITLE
Logging format needs improvement

### ISSUE THREAD

**Issue by IndustrialOne** [uTox/uTox#1043]
The text file that my chat log is saved in is not readable in notepad. If the format you're saving it is in plaintext then everything in it should be plaintext cuz I have no idea what to open it with to see the correct timestamps and not random characters. Also, there is no colon or space after my name that precedes the line of text.

Here's the mess the chatlog currently looks like:

<�ŸY�
� SplorpLookin ass
� {�ŸY� � � SplorpThis is way fucking better dude
� €�ŸY� � � SplorpShit looks nice
� ‡�ŸY� � � SplorpTheres no stupid hidden buttons
� ‹�ŸY� � � SplorpShits not lagging at all
� š�ŸY� � SplorpLooks better than skype actually
�
It needs to look more like Skype's:

[1/23/2016 10:57:50 AM] Splorp: not a whole lot
[1/23/2016 10:57:57 AM] Splorp: but at least 1/3 of the buds are covered in purple
[1/23/2016 4:48:23 PM] Splorp: yo we should try playing some mario party today or tomorrow

**Comment by robinlinden** [comment]
Yeah, currently for them to be readable plaintext you have to export them from inside the client. Right click the contact and there should be an option for it in the contact settings.

The files you're looking at have control characters telling uTox e.g. how long the username is, if the contact you sent the message to sent an ack back so we know if we should resend it or not and things like that.
